apps/server/src/modules/news/controller/dto/update-news.params.ts

```diff
@@ -1,4 +1,5 @@
 import { ApiPropertyOptional } from '@nestjs/swagger';
+import { SanitizeHtml } from '@shared/controller';
```

Comment:
Only style but external inputs on top, internal after it?

Reply:
Yes, but isn't `'@shared/controller'` internal?